src/System.Management.Automation/engine/ExecutionContext.cs

```diff
@@ -1258,14 +1261,20 @@ internal static Assembly LoadAssembly(string name, string filename, out Exceptio
                 catch (BadImageFormatException badImage)
                 {
                     error = badImage;
+                    return null;
```

Comment:
The `return null` at this line and line 1269 are not needed.

Reply:
resolved